PS8lite.Rmd

```diff
@@ -187,23 +187,37 @@ MiscVal + MoSold + YrSold" %>%
 Fit one Random Forest model `model_rf` using any approach you want.
 
 ```{r}
-
+# Fit random forest on training data:
+model_rf <- randomForest(
+  form = model_formula,
+  data = training, 
+  mtry = 2,
+  ntree = 100 
+)
 ```
 
-
-
 ***
 
-
-
 # Make predictions and submit to Kaggle
 
 Using `model_rf` make predictions and submit to Kaggle.
 
 ```{r}
+# Predict on test data
+test <- test %>% 
+  mutate(
+    logSalePrice_hat = predict(object = model_rf, newdata = test),
+    SalePrice_hat = exp(logSalePrice_hat) -1
+  )
+```
 
+```{r, eval=FALSE}
+submission_rf <- test %>%
+  mutate(SalePrice = SalePrice_hat) %>%
+  select(Id, SalePrice)
+write_csv(submission_rf, "submission_rf.csv")
```

Comment:
Hi, everything looks good. Here is an example comment that I would leave as part of a “code review” if this were real life. Then you would make the requested changes to this PR and then I would merge your changes into my master copy of the repo. I won't be doing that last step in this case.